tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/varref/ErrorVariableReferenceTest.java

```diff
@@ -38,6 +38,8 @@
 public class ErrorVariableReferenceTest {
 
     private CompileResult result;
+    final String errorCauseWithWildcardBPDlog =  "cannot use a wildcard binding pattern to bind the cause: a wildcard binding " +
```

Comment:
Please define as a constant and use the proper access modifier. Also please follow naming conventions.

```suggestion
    private static final String INVALID_WILDCARD_BP_BINDING_ERROR_CAUSE_ERROR =  "cannot use a wildcard binding pattern to bind the cause: a wildcard binding " +
```

Reply:
[d923b81](https://github.com/ballerina-platform/ballerina-lang/pull/39691/commits/d923b81d7a28fc11f177f2f505e317c627ea8870)